create_box/main.py

```diff
@@ -1,9 +1,29 @@
 """This is the entry point of the program."""
 
+def create_box(height, width, character, solid_box=True):
+    
+    # Test for invalid input.
+    if height < 1 or width < 1:
+        raise Exception('ArgumentError', 'Height and width must be greater than 0.')
+    
+    new_string = ""
+    
+    # Create the box.
+    for line in range(height):
 
-def create_box(height, width, character):
-    pass
+        # Test for border box conditions...
+        if not solid_box and line > 0 and line < (height - 1) and width > 2:
```

Comment:
The `width > 2` check here isn't necessary. Your `' '*(width-2)` code on the next line will give a zero-length string if the width is less than three, which will still result in the appropriate output.